spec/src/main/asciidoc/chapters/controllers.asciidoc

```diff
@@ -177,7 +177,7 @@ public class MyController {
 
     @GET
     public String get() {
-        return "mybean.jsp"; // mybean.value accessed in JSP
+        return "mybean.jsp"; // mybean.value accessed in Jakarta Server Pages
```

Comment:
Maybe instead:

> mybean.value accessed in Jakarta Server Pages **page**

Or 

> mybean.value accessed in Jakarta Server Pages **file**


This also sounds weird. I guess the problem is that we often used the term "JSP" to refer to a JSP file and not to the spec itself.

Reply:
Maybe simply use Java Server Page when referring to a single file and Java Server Pages when referring to the technology?

Reply:
Yes, great idea!